scripts/drag-n-bar.js

```diff
@@ -1438,6 +1450,13 @@ H5P.DragNBar.prototype.findNewPoint = function (originX, originY, angle, distanc
     }
     frame.rotate = angle;
 
+    // set start transform
+    const transformCSSTranslateXYArray = $element[0].style.transform.split("px");
+    const transformCSSTranslateX = parseInt(transformCSSTranslateXYArray[0].match(/-?\d+/g));
+    const transformCSSTranslateY = parseInt(transformCSSTranslateXYArray[1].match(/-?\d+/g));
```

Comment:
This works, however passing an array into `parseInt` might give unexpected results:

```js
parseInt(["123"]) === 123;
parseInt(["123", "321"]) === 123;
```

I'd prefer to use the first element of the match array:

```suggestion
    const transformCSSTranslateX = parseInt(transformCSSTranslateXYArray[0].match(/-?\d+/g)[0]);
    const transformCSSTranslateY = parseInt(transformCSSTranslateXYArray[1].match(/-?\d+/g)[0]);
```

Reply:
Aha, TIL